mkosi/config.py

```diff
@@ -56,18 +56,21 @@ def parse_boolean(s: str) -> bool:
     die(f"Invalid boolean literal: {s!r}")
 
 
-def parse_path(value: str, *, required: bool, absolute: bool = True, expanduser: bool = True, expandvars: bool = True) -> Path:
+def parse_path(value: str,
+               *,
+               required: bool = True,
+               absolute: bool = True,
+               expanduser: bool = True,
+               expandvars: bool = True) -> Path:
     if expandvars:
         value = os.path.expandvars(value)
 
     path = Path(value)
 
     if expanduser:
-        user = current_user()
-        if path.is_relative_to("~") and not user.is_running_user():
-            path = user.home / path.relative_to("~")
-        else:
-            path = path.expanduser()
+        if path.is_relative_to("~") and not InvokingUser.is_running_user():
+            path = InvokingUser.home() / path.relative_to("~")
+        path = path.expanduser()
```

Comment:
`InvokingUser.is_running_user()` and `InvokingUser.home()` are only used here, so should we instead do:

```suggestion
        path = InvokingUser.expanduser(path)
```